src/api/zoops.ts

```diff
@@ -0,0 +1,49 @@
+import express from "express";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+const zoopsRouter = express.Router();
+
+// GET /api/zoops
+
+zoopsRouter.get("/", async (req, res, next): Promise<void> => {
+    try {
+        const zoops = await prisma.zoop.findMany();
+        res.send({zoops});
+    } catch (e) {
+        next(e);
+    }
+})
+
+// POST /api/zoops
+
+zoopsRouter.post("/", async (req, res)=> {
+        const {content, authorId, receiverId} = req.body;
+        const zoop = await prisma.zoop.create({
+            data: {
+                content,
+                authorId,
+                receiverId
+            }
+        })
+        res.send({zoop});
+})
+
+// PUT /api/zoops/:id
+//TODO: add auth middleware function when auth.ts file is available for import
+zoopsRouter.put("/:id", async (req, res, next)=> {
```

Comment:
nit: It feels like this is really a PATCH since it is just updating the content? but it's up to you

Reply:
I'll make an issue to change it if there is time. I only did PUT because that was what was in the Zoop requirements doc.